src/main/java/org/broadinstitute/consent/http/service/DatabaseElectionAPI.java

```diff
@@ -451,59 +458,56 @@ private boolean validateAllDatasetElectionsAreClosed(List<Election> elections){
 
     private Election validateAndGetDULElection(String referenceId, ElectionType electionType) throws Exception {
         Election consentElection = null;
-        if(electionType.equals(ElectionType.DATA_ACCESS)){
-            Document dar = describeDataAccessRequestById(referenceId);
-            if(dar == null){
+        if (electionType.equals(ElectionType.DATA_ACCESS)) {
+            DataAccessRequest dataAccessRequest = dataAccessRequestService.findByReferenceId(referenceId);
+            if (Objects.isNull(dataAccessRequest)) {
                 throw new NotFoundException();
             }
-            List<DataSet> dataSets = verifyDisableDataSets(dar, referenceId);
+            List<DataSet> dataSets = verifyActiveDataSets(dataAccessRequest, referenceId);
             Consent consent = consentDAO.findConsentFromDatasetID(dataSets.get(0).getDataSetId());
-            consentElection = electionDAO.findLastElectionByReferenceIdAndStatus(consent.getConsentId(), "Closed");
-            if((consentElection == null)){
-                throw new IllegalArgumentException(DUL_NOT_APROVED);
-            } else {
-                Integer openElections = electionDAO.verifyOpenElectionsForReferenceId(consent.getConsentId());
-                Vote vote = voteDAO.findVoteByElectionIdAndType(consentElection.getElectionId(), VoteType.CHAIRPERSON.getValue());
-                if((openElections != 0) || (vote != null && !vote.getVote())) {
-                    throw new IllegalArgumentException(DUL_NOT_APROVED);
-                }
-            }
```

Comment:
This is essentially a new feature - we don't want to fail DAR election creation just because there are no consent elections for the associated dataset.